source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -194,15 +207,37 @@ A driver-defined wrapper around a single TCP/IP connection to an Endpoint. A Con
        *  when this connection was created
        */
       generation: number;
+
+      /**
+       * The current state of the Connection.
+       *
+       * Possible values are the following:
+       *   - "unestablished": The Connection is empty. It has been created but has not opened a socket or
+       *                      performed any I/O. Contributes to totalConnectionCount.
```

Comment:
I suggest avoiding negatives ("un"). Also, what is the state for when a socket was created but before the connection is established?

Possible wording:

> "pending": The Connection has been created but it is not yet established. Contributes to totalConnectionCount.